src/seeders/20200103073111-users-seed-file.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @exports
+ * @class
+ * @param {object} queryInterface
+ * @param {object} Sequelize
+ */
+export function up(queryInterface) {
+  return queryInterface.bulkInsert('Users', [{
+    firstName: 'James',
+    lastName: 'Bond',
+    email: 'jamesbond@gmail.com',
+    password: process.env.SUPER_ADMIN_PASSWORD,
```

Comment:
Why are you passing the password as an environment variable? All data should be auto-generated using faker

Reply:
@adesege  Here its in the seed file. Once a dummy seed admin user is inserted, i will need to log him in and get the token that will allow him to use /reset-role route. if i use faker, i will not be able to know his email and password. I only used faker for the fixtures in test

Reply:
Then you should create the user before the test runs. Check other tests that use the before hook

Reply:
@adesege This user here is not for mocha test. its for dev environment for `POSTMAN`.
For `mocha test`, i have generated a faker admin user data except few fields.

Reply:
I would rather have you manually create a user and assign the super_admin role to it than using a seeder file to do that. This have some security implications to it. This seeder file can be run in production and your password hash should not leave your database.

Reply:
@adesege Ok, then i am gonna prevent the application to create this admin user in dev environment and add this user manually in the DB.